fe/fe-core/src/main/java/org/apache/doris/nereids/processor/pre/TurnOffPipelineForDml.java

```diff
@@ -53,6 +58,15 @@ private void turnOffPipeline(StatementContext context) {
         // set temporary session value, and then revert value in the 'finally block' of StmtExecutor#execute
         sessionVariable.setIsSingleSetVar(true);
         try {
+            StatementBase parsedStatement = context.getParsedStatement();
```

Comment:
not change this file, u should add a new file to do this thing

Reply:
done